inkscape/src/main/assembly/dist/lace_grid.inx

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
 
-	<_name>Lace Grid</_name>
+	<_name>Rectangular Grid</_name>

```

Comment:
To make matters worse: diamond would be better than "standard" as first style of the polar grids and  I suppose we could add the same style options to your grid. So we need something more generic at the menu-level. Perhaps regular grid is an option.

By its nature, a wiki page is not carved in stone, so that can be changed too. I interpreted/intended rectangular as "filling a rectangular space" to create rectangular mat, as opposed to polar for edgings on round mats. 

Inscape uses rectangular and axonometric on the document properties dialog. Under extension-render-grids we have: cartesian, isometric, polar and no-adjective. Lacemakers also use honeycomb. I guess hexagonal sits between honeycomb and axonometric/isometric on the familiarity scale, but that might be a personal bias.